config/index.js

```diff
@@ -46,7 +46,7 @@ async function getHashiConfig(configPath) {
 			extendsConfig = await getHashiConfig(extendsConfigPath)
 		}
 
-		if (process.env.VERCEL !== 'production') {
+		if (process.env.VERCEL_ENV !== 'production') {
```

Comment:
This is a typo I found which might be why hcp previews weren't working correctly in UDR. 🤷🏻